src/AccessibilityInsights.SharedUx/Resources/Styles.xaml

```diff
@@ -504,6 +504,9 @@
         <Setter Property="Foreground" Value="{DynamicResource ResourceKey=PrimaryFGBrush}" />
     </Style>
     <Style x:Key="lviTabStops" TargetType="{x:Type ListViewItem}">
+        <Setter Property="FocusVisualStyle" Value="{DynamicResource {x:Static SystemParameters.FocusVisualStyleKey}}"/>
+        <Setter Property="BorderBrush" Value="Transparent"/>
+        <Setter Property="BorderThickness" Value="2,0"/>
```

Comment:
I needed to shift the contents to the right to make room for the focus indicator. I moved everything to the right to make the focused item line up better with the non-focused items.